components/tikv_alloc/Cargo.toml

```diff
@@ -14,31 +14,36 @@ jemalloc = ["jemallocator", "jemalloc-sys", "jemalloc-ctl"]
 mem-profiling = ["jemallocator/profiling"]
 
 [dependencies]
+failure = "0.1.5"
 libc = "0.2"
 log = "0.3"
 
 [dev-dependencies]
 tempdir = "0.3"
 
+# FIXME: Shouldn't need to depend on the git repos but jemalloc-ctl 0.3
+# isn't published.
+#
+# https://github.com/gnzlbg/jemallocator/issues/113
+
 [dependencies.jemallocator]
-version = "0.1.9"
+version = "0.3.0"
```

Comment:
Is there any special reason for `0.3.0` instead of `0.3`? I guess the last one is the common usage.

Reply:
It's my habit. It indicates to me that that is the exact version that was current and tested when I wrote it.